### PR TITLE
Style

### PR DESCRIPTION
**Pull request by zusoomro**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.DS_Store
```
Comment (allysmith12) [inline .gitignore:1]: Oh thank god.

File: mobile/src/shared/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import {
```
Comment (allysmith12) [inline mobile/src/shared/Button.tsx:1]: Is this just a fancy shared button??

**Review by allysmith12 — APPROVED** [review]
Sure, LGTM

File: mobile/src/shared/Button.tsx (hunk 2 above)
Comment (zusoomro) [reply mobile/src/shared/Button.tsx:1]: Yea it's easier to style

**Comment by zusoomro** [comment]
This is the most recent version (that I demo'd) and has a bunch of changes. The branch name isn't representative of the work here. The changes can be viewed in the commit descriptions as well, but here's a high level list

- Design and style changes across the front end
- Use the same component for login and register for the time being
- Use the same component to create and modify an event
- Pods now have members, which are all the users in that pod

**Comment by chloeprezelski** [comment]
Is there a reason this hasn't been merged yet?

**Comment by allysmith12** [comment]
Zulfi can we merge this so I can merge in my stuff please :)

**Pull request merged**